package.json

```
@@ -25,6 +25,18 @@
     "check:unused-exports": "ts-unused-exports tsconfig.json --excludePathsFromReport=packages/types/ --excludePathsFromReport=packages/server/src/types.ts --ignoreFiles=/dist/",
     "codegen": "lerna run --stream codegen",
     "deploy:images": "lerna run --stream deploy:image",
+    "docker:build": "run-s \"docker:build:*\"",
+    "docker:build:server": "docker build -f packages/server/docker/server/Dockerfile -t api-server-local-build:latest .",
+    "docker:build:swr-queue-worker": "docker build -f packages/server/docker/swr-queue-worker/Dockerfile -t api-swr-queue-worker-local-build:latest .",
+    "docker:rm": "run-p --continue-on-error --print-label \"docker:rm:*\" ",
```

```suggestion
    "docker:rm": "docker image rm api-server-local-build api-swr-queue-worker-local-build",
```
Is this doing the same?
If yes, then I would find it easier to read because I don't have to know about run-p and have read the lines below.

(The same question applies to docker run and docker stop scripts below.)

---

> Is this doing the same?

Yes

> If yes, then I would find it easier to read because I don't have to know about run-p and have read the lines below.

Then it would be duplicated code + it is wise to know about `npm-run-all` since we use it in all repos

> (The same question applies to docker run and docker stop scripts below.)

It is not possible to combine the run commands in an easy way...

---

> Then it would be duplicated code

I am viewing this more as a list to look up the commands I cannot remember than actual code but I can also follow that reasoning.